util/generate-sample-ledger/src/lib.rs

```diff
@@ -44,85 +44,60 @@ pub fn bootstrap_ledger(
         num_outputs, picomob_per_output
     );
 
-    let mut blocks_and_transactions: Vec<(Block, Vec<RedactedTx>)> = Vec::new();
+    let mut blocks_and_contents: Vec<(Block, BlockContents)> = Vec::new();
     let mut previous_block: Option<Block> = None;
 
+    let mut rng: FixedRng = SeedableRng::from_seed([33u8; 32]);
+
     for block_index in 0..num_blocks as u64 {
         println!("Creating block {} of {}.", block_index, num_blocks);
 
-        // Transactions in this block.
-        let minting_transactions: Vec<RedactedTx> = recipients
-            .par_iter()
```

Comment:
do you know what the `par_iter` thing here was? i think it was to try to make bootstrap faster because it can be really slow when the numbers are big i think

Reply:
I added that at one point to speed things up, but I think the real problem was that I had forgotten to compile bootstrap in release mode.